app/models/user.rb

```diff
@@ -56,6 +56,19 @@ def send_activation_email
     UserMailer.account_activation(self).deliver_now
   end
 
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```

Comment:
```suggestion
    update_columns  reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```
method naỳ mới hợp lý nhất cu nhé

Reply:
vâng ạ